assets/js/components/sell-quick-start.component.js

```diff
@@ -144,6 +150,8 @@ function sellQuickStartController ($scope, $rootScope, currency, buySell, Alerts
       if (r.absoluteFeeBounds[0] === 0) {
         $scope.error['moreThanInWallet'] = true;
         $scope.offerUseAll();
+      } else {
+        $scope.status = {};
```

Comment:
Why just reset `status` here? If `r.absoluteFeeBounds[0] === 0`, then busy will stay set to true.